optuna/_gp/acqf.py

```diff
@@ -7,8 +7,11 @@
 
 import numpy as np
 
-from optuna._gp.gp import GPRegressor
-from optuna._gp.search_space import SearchSpace
+
+if TYPE_CHECKING:
+    from optuna._gp.gp import GPRegressor
+    from optuna._gp.search_space import SearchSpace
+
```

Comment:
The `TYPE_CHECKING` section is already prepared below, so could you please move these module imports to the section?